packages/bot-web-ui/src/utils/datadog-logs.ts

```diff
@@ -9,9 +9,14 @@ import { formatDate, formatTime } from '@deriv/shared';
  * @returns {void}
  * **/
 const initDatadogLogs = (is_datadog_enabled: boolean) => {
-    if (!is_datadog_enabled) {
-        if (window.DD_RUM) {
-            window.DD_RUM = undefined;
+    if (is_datadog_enabled) {
```

Comment:
put some comments on this so others can understand it later

Reply:
Alright @adrienne-deriv, this simply means that if `datadog` is `enabled` in Firebase and `datadogLogs` is already initialised, we `"grant"` tracking consent and `return` from the `function`. This prevents re-initialisation, which would otherwise cause an error. If `datadogLogs` is not initialised yet, we continue execution further down the function to initialise it.

On the other hand, if `datadog` is not `enabled` in Firebase, we check if `datadogLogs` is already initialised. If it is, we revoke the tracking consent and `return` from the `function`. If it's not initialised yet, then we still just `return` from the `function`